src/dex-ui-components/SwapTokens/types.ts

```diff
@@ -5,13 +5,27 @@ export interface SwapTokensState {
     displayAmount: string;
     balance: number | undefined;
     poolLiquidity: number | undefined;
+    tokenName: string | undefined;
```

Comment:
we might not need all this token Info in the app also if we do we need to make action which can update the rest values while changing a token symbol so as of now I am keeping all till our final discussion.